packages/shared/src/components/auth/OnboardingSteps/ContentTypes/CustomCheckbox.tsx

```diff
@@ -0,0 +1,47 @@
+// TODO: move in a new component in shared if the onboardingContentType experiment is successful
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import { Checkbox } from '../../../fields/Checkbox';
+
+interface CustomCheckboxProps {
+  checked: boolean;
+  title: string;
+  description: string;
+  onCheckboxToggle: () => void;
+  name: string;
+}
+export const CustomCheckbox = ({
```

Comment:
Styling kept almost the same as the checkbox created for layout experiment in onboarding